selfdrive/car/hyundai/carstate.py

```diff
@@ -32,12 +32,18 @@ def __init__(self, CP):
     self.park_brake = False
     self.buttons_counter = 0
 
+    # When available we use CLU15->CF_Clu_VehicleSpeed2 to populate vEgoCluster which is the actual dash speed
+    # However, on some cars this is not always present, so we use the less accurate CF_Clu_VehicleSpeed signal
+    self.accurate_dash_speed_seen = False
+
```

Comment:
The main difference is that `CF_Clu_VehicleSpeed` can sometimes be slightly noisier and the timing of when it increments or decrements speed is slightly different than the dash. However, in a steady state it should match the dash much better than vEgo as it includes the same dash factor as `CF_Clu_VehicleSpeed2`.

Reply:
I chose to remove `CF_Clu_VehicleSpeed` from this PR, as it can rapidly oscillate by a full 1 kph sometimes on certain cars which can't be removed with hysteresis, as it seems the signal already has hysteresis (makes steady state inaccurate). `CF_Clu_VehicleSpeed2` affects a lot of cars so I think this PR is good to go as is, once we get a few testers